### DISCUSSION TITLE
React Native - IOS: Archive Failed - No Such Module

### DISCUSSION THREAD

**Discussion by devilankur18** [codemagic-ci-cd/codemagic-docs#1153] [Q&A] [answered]
When building for react native IOS, the build is failing very strangely. Pod Install have worked, but somehow its not starting to build. This build is happening from the UI though, but not working from CLI.
 

![image](https://user-images.githubusercontent.com/431165/201462799-4cf4ae6b-813d-451c-8eff-7d470e83742d.png)

Build details - https://codemagic.io/app/6361335164db23ea9241e3f3/build/636a1bbe030e6bcce2107efa

**Comment by mikhail-tokarev** [comment]
one possible reason you don't have `use_frameworks!` in your Podfile

You can find an example here https://cocoapods.org/pods/CTNotificationService#cocoapods

**Reply by devilankur18** [reply]
@mikhail-tokarev its already there in the Podfile. Also this build is happening from the Xcode UI, just not working with CLI.

**Comment by himesh-cm** [comment]
@devilankur18 Thanks for sharing the build info, I will check and get back to you.

**Comment by icarusdust** [comment, marked as answer]
Had a chance to check the logs. Please try building it on your workspace rather than the project. Meaning that you need to use  `--workspace` instead of `--project` in the build command. Let me know if it helps.

**Comment by devilankur18** [comment]
@icarusdust thanks it helped.
